CRM/Civicase/APIHelpers/CasesByContactInvolved.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+class CRM_Civicase_APIHelpers_CasesByContactInvolved {
+  /**
+   * Adds joins and conditions to the given query in order to filter cases by
+   * contact involvement.
+   *
+   * @param CRM_Utils_SQL_Select $query the SQL object reference.
```

Comment:
Please add the description for the param after the @param line.  Something like this:
```php
@param CRM_Utils_SQL_Select $query 
  The SQL object reference.
```